woocommerce-pdf-invoices-packingslips.php

```diff
@@ -452,6 +453,51 @@ public function rtl_detected() {
 			}
 		}
 	}
+	
+	/**
+	 * Yearly reset action missing notice
+	 * 
+	 * @return void
+	 */
+	public function yearly_reset_action_missing_notice(): void {
+		if ( ! $this->settings->maybe_schedule_yearly_reset_numbers() ) {
+			return;
+		}
+		
+		if ( ! function_exists( 'as_get_scheduled_actions' ) ) {
+			return;
+		}
+		
+		$current_date   = new \DateTime();
+		$end_of_year    = new \DateTime( 'last day of December' );
+		$days_remaining = $current_date->diff( $end_of_year )->days;
+
+		// Check if the current date is within the last 30 days of the year
+		if ( $days_remaining <= 30 && ! $this->settings->yearly_reset_action_is_scheduled() ) {
+			ob_start();
```

Comment:
Why using buffer and not just using echo?

Reply:
I think all notices are using it, we could improve in a new issue.